README.md

```diff
@@ -101,6 +101,18 @@ Arguments:
                 Must be >= 33792 kb (~33 mb)
   --label       Label of the new image, defaults to "RUNTIMEJS"
 ```
+### Environment Variables*
+There is currently no officially supported process for passing environment variables into a runtime instance. There is a work around that will allow the passing of Kernel Command Line arguments instead.
```

Comment:
Probably a good idea to add a link to the issue on the main repo (https://github.com/runtimejs/runtime/issues/134) here (e.g. "While there is a \[proposal](link-here) to add this feature, there is currently no officially...").